en/app.ftl

```diff
@@ -330,6 +338,22 @@ premium-promo-use-cases-gaming-body-2 = Worried about other gamers or trolls lea
 
 premium-promo-pricing-free-price = Your current Free plan
 
+## The Premium waitlist page
+
+waitlist-heading = Join the { -brand-name-relay-premium } Waitlist
+waitlist-lead = Get notified when { -brand-name-firefox-relay-premium } is available for your region.
+waitlist-control-required = Required
+waitlist-control-email-label = What is your email address?
+waitlist-control-email-placeholder = yourname@example.com
+waitlist-control-country-label = What country do you live in?
+waitlist-control-locale-label = Select your preferred language.
+waitlist-submit-label = Join the Waitlist
+waitlist-privacy-policy-agree = By clicking “Join the waitlist”, you agree to our <a>Privacy Policy</a>.
```

Comment:
Nit: the label has Waitlist uppercase, assuming it's the one above?

Reply:
Wow, you really do have a sharp eye! :)

![image](https://user-images.githubusercontent.com/4251/165750501-a2ed77e5-6e54-4c75-8350-25d3e9eb1463.png)

What do you think of the approach in 4f64596, where I just pull the correct value from a variable?